.github/workflows/publish-common-library-to-npm.yml

```diff
@@ -20,6 +20,24 @@ jobs:
         with:
           sparse-checkout: 'Common'
           sparse-checkout-cone-mode: false
+        
+      - id: get-package-json
+        run: |
+          package_content=`cat package.json`
+          echo "json<<EOF" >> $GITHUB_OUTPUT
+          echo $package_content >> $GITHUB_OUTPUT
+          echo 'EOF' >> $GITHUB_OUTPUT
+
+      - uses: actions/github-script@v5
+        with:
+          script: |
+            github.rest.git.createRef({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              ref: 'refs/tags/lib-common-v${{ fromJson(steps.get-package-json.outputs.json).version }}',
```

Comment:
Can you remove the `v` from this?

It is not valid semver, which means the GH auto releases won't find the right range.

https://semver.org/#is-v123-a-semantic-version